src/components/data_validation_overlay/dv_checkbox/dv_checkbox.ts

```diff
@@ -13,6 +13,7 @@ css/* scss */ `
     height: ${CHECKBOX_WIDTH}px;
     accent-color: #808080;
     margin: ${MARGIN}px;
+    position: absolute;
```

Comment:
I don't see a difference (even in firefox), but I'm gonna trust you that there was actually  an issue and that this line does something.

I'd add a comment to explain why it's needed, because in 3 months that'll be CSS we remove because we do'nt see what it does.

Reply:
it's only visible when iin Odoo, where the font-size is set to 14px (instead of 12 in our demo sheet).  You can convince yourself by editing the line in `demo/main.css` to 17px for exampl,e itd become clear as day ^^